keras_cv/models/backbones/resnet_v1/resnet_v1_backbone_presets.py

```diff
@@ -0,0 +1,128 @@
+            ),
+        },
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [2, 2, 2, 2],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "basic_block",
+        },
+    },
+    "resnet34": {
+        "metadata": {
+            "description": (
+                "ResNet model with 34 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style)."
+            ),
+        },
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [3, 4, 6, 3],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "basic_block",
+        },
+    },
+    "resnet50": {
+        "metadata": {
+            "description": (
+                "ResNet model with 50 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style)."
+            ),
+        },
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [3, 4, 6, 3],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "block",
+        },
+    },
+    "resnet101": {
+        "metadata": {
+            "description": (
+                "ResNet model with 101 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style)."
+            ),
+        },
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [3, 4, 23, 3],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "block",
+        },
+    },
+    "resnet152": {
+        "metadata": {
+            "description": (
+                "ResNet model with 152 layers where the batch normalization "
+                "and ReLU activation are applied after the convolution layers "
+                "(v1 style)."
+            ),
+        },
+        "config": {
+            "stackwise_filters": [64, 128, 256, 512],
+            "stackwise_blocks": [3, 8, 36, 3],
+            "stackwise_strides": [1, 2, 2, 2],
+            "include_rescaling": True,
+            "input_shape": (None, None, 3),
+            "input_tensor": None,
+            "block_type": "block",
+        },
+    },
+}
+
+backbone_presets_with_weights = {
```

Comment:
Not so much an issue for this PR, but something for us to think about (also @jbischof)

If we want weight versioning (e.g. new imagenet weights when we improve our training scripts), does this become a new preset? How do we name this / by default give people the latest and greatest weights?

In the old weight loading system we did this by defaulting to the newest weights when `weights="imagenet"` and only using older versions if a `-v0` suffix was included

Reply:
I do not have a good idea on this issue. Let's wait for @jbischof .

Reply:
@ianstenbit the pattern we use for KerasNLP is to version the filenames but not the presets ([example](https://github.com/keras-team/keras-nlp/blob/1b1781fc195380e529f862c80521a4518f7ec6d3/keras_nlp/models/bert/bert_presets.py#L72)). If a superior weights file becomes available, we add the new file to the preset and leave the old file in GCP so that old versions of the package continue to function. WDYT?

Reply:
Fine with me @jbischof, but it does prevent users from being able to pin down a specific weight version if they like reproducibility/dont want anything to change out from under them.  

I don't think thats compelling enough to support old weight versions (at least unless we have someone specifically asking for it), but a consideration.

Reply:
In that case we might want to offer versioned presets names. A little awkward for this case because we already have a version number, tho.